README.rst

```diff
@@ -110,7 +110,7 @@ messages to it
         project_id=os.getenv('GOOGLE_CLOUD_PROJECT'),
         topic='MY_TOPIC_NAME',  # Set this to something appropriate.
     )
-    publisher.create_topic(topic_name)
+    publisher.create_topic(request={"name": topic_name})
```

Comment:
The signature here is fine - this particular method supports subscription name and topic name as keyword arguments ([source](https://github.com/googleapis/python-pubsub/blob/7228f6c9a4c050bf22bb4bc3582b89b04eaa8702/google/pubsub_v1/services/publisher/client.py#L277-L285)).